packages/wallet-connect-sdk-svelte/README.md

```diff
@@ -0,0 +1,51 @@
+# WalletConnect SDK Svelte
+
+- [Installation](#installation)
+- [Setup](#setup)
+- [Usage](#usage)
+
+## Installation
+
+Install the dependencies on your application
+
+### NPM
+
+```
+npm i @cityofzion/wallet-connect-sdk-svelte @walletconnect/sign-client @walletconnect/types
+```
+
+### YARN
+
+```
+yarn add @cityofzion/wallet-connect-sdk-svelte @walletconnect/sign-client @walletconnect/types
+```
+
+## Setup
+To begin development, first establish an account on the [Wallet Connect website](https://walletconnect.com/). Next,
+generate a new Project. This process is straightforward, requiring only a few form fields.
+
+### Instantiate an object of the WCSDKStore class
+
+```ts
+import { WCSDKStore } from '@cityofzion/wallet-connect-sdk-svelte'
+
+const wcOptions = {
+    projectId: '<your wc project id>', // the ID of your project on Wallet Connect website
+    relayUrl: 'wss://relay.walletconnect.com', // we are using walletconnect's official relay server
+    metadata: {
+        name: 'MyApplicationName', // your application name to be displayed on the wallet
+        description: 'My Application description', // description to be shown on the wallet
+        url: 'https://myapplicationdescription.app/', // url to be linked on the wallet
+        icons: ['https://myapplicationdescription.app/myappicon.png'] // icon to be shown on the wallet
+    }
+};
+
+<script>
```

Comment:
The script tag should be on the beginning of the code, right?

Reply:
not necessarily, there may be other imports before it